JetNews/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Thu Feb 27 20:17:30 CET 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-rc-1-all.zip
```

Comment:
6.3 is out (i.e. no need for RC) and we tend to use `bin` releases.

Reply:
Ah, didn't know it was out. Just went with what AS told me to in the quickfix. Will update and switch to bin. Out of curiosity, any reason why you use `bin`? AFAIK `all` is preferred for usage in IDEs, and AS itself suggests to switch to `all`.

Reply:
IIRC it's just because it's a smaller download if you don't need source.